srcpkgs/broot/template

```diff
@@ -1,6 +1,6 @@
 # Template file for 'broot'
 pkgname=broot
-version=1.17.1
+version=1.18.0
```

Comment:
1.19 available

Reply:
1.19 requires Rust 1.65. I've reported it in the [tracking issue](https://github.com/void-linux/void-packages/issues/40805) as well.

Reply:
aha, np